docs/source/learn/core_notebooks/pymc_aesara.ipynb

```diff
@@ -0,0 +1,1960 @@
+{
```

Comment:
<p>There's a missing <code>-</code> in <code>np.exp(10)</code> in the scipy call, that's why they don't match.</p><p> </p><p>Something is also odd with the sigma logp, it shouldn't match with the scipy because of the jacobian term (which should just be a -10)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='106'/>

Reply:
Thanks! I added a comment about it. I do believe is the Jacobian.

Reply:
<p>Ah awesome, I missed the -10 at the beginning, I was looking at the end xD
</p><p> </p><p>World order is restored!</p>